Dockerfile

```diff
@@ -450,6 +451,12 @@ RUN echo "BUILD_BRANCH=${BUILD_BRANCH}"    | tee /build.info \
     && echo "FDD_LENSES_CLI_VERSION=${LC_VERSION}" | tee -a /build.info \
     && sed -e 's/^/FDD_/' /opt/landoop/build.info  | tee -a /build.info
 
+# Add man pages & README.txt
+ADD ./filesystem/usr/local/man /usr/local/share/
```

Comment:
You don't have to this, we COPY all the filesystem.
https://github.com/Landoop/fast-data-dev/blob/master/Dockerfile#L374